openquakeplatform/openquakeplatform/static/js/irv_viewer.js

```diff
@@ -14,1043 +14,839 @@
-                tmp = 0;
-                for(var k in iriIndicator) {
-                    getNewValues(k);
-                    tmp = (sviValue * aalValue);
-                    iriIndicator[k] = tmp;
+    $('#map-tools').append(
+        '<select id="svir-project-list">'+
+            '<option selected disabled>Select Project</option>'+
+        '</select>'
+    );
+
+    $('#map-tools').append(
+        '<select id="thematic-map-selection">'+
+            '<option>Select Indicator</option>'+
+        '</select>'
+    );
+
+    $('#thematic-map-selection').css({ 'margin-bottom' : 0 });
+    $('#svir-project-list').css({ 'margin-bottom' : 0 });
+    $('#region-selection-list').css({ 'margin-bottom' : 0 });
+    $('#svir-project-list').hide();
+    $('#region-selection-list').hide();
+    var SVIRLayerNames = [];
+    var url = "/geoserver/ows?service=WFS&version=1.0.0&REQUEST=GetCapabilities&SRSNAME=EPSG:4326&outputFormat=json&format_options=callback:getJson";
+
+    // Get layers from GeoServer and populate the layer selection menu
+    $.ajax({
+        url: url,
+        contentType: 'application/json',
+        success: function(xml) {
+            //convert XML to JSON
+            var xmlText = new XMLSerializer().serializeToString(xml);
+            var x2js = new X2JS();
+
+            var jsonElement = x2js.xml_str2json(xmlText);
+            var featureType = jsonElement.WFS_Capabilities.FeatureTypeList.FeatureType;
+
+            // Find the SVIR keywords
+            var stringToLookFor = 'SVIR_QGIS_Plugin';
+            for (var i = 0; i < featureType.length; i++) {
+                if (featureType[i].Keywords.indexOf(stringToLookFor) > -1) {
+                    SVIRLayerNames.push(featureType[i].Title + " (" + featureType[i].Name + ")");
                 }
             }
+            // Append the layer to the selection dropdown menu
+            for (var ij = 0; ij < SVIRLayerNames.length; ij++) {
+                $('#svir-project-list').append('<option>'+ SVIRLayerNames[ij] +'</option>');
+            }
+            $('#svir-project-list').show();
+        },
+        error: function() {
+            $('#ajaxErrorDialog').empty();
+            $('#ajaxErrorDialog').append(
+                    '<p>This application was not able to get a list of layers from GeoServer</p>'
+                );
+            $('#ajaxErrorDialog').dialog('open');
+        }
+    });
 
-            ///////////////
-            //// Scale ////
-            ///////////////
+    // Get the layer metadata (project def)
+    $('#svir-project-list').change(function() {
+        $('#projectDef-spinner').show();
+        $('#iri-spinner').show();
+        $('#regionSelectionDialog').empty();
+        // Get the selected layer
+        var selectedLayer = document.getElementById('svir-project-list').value;
+        // clean the selected layer to get just the layer name
+        selectedLayer = selectedLayer.substring(selectedLayer.indexOf("(") + 1);
```

Comment:
Please add a FIXME comment here, specifying that this doesn't work in case the project title contains any parentheses